---
title: Learning Causal Representations of Single Cells via Sparse Mechanism Shift
  Modeling
section: Poster
openreview: IOWJsPJ2xGd
abstract: Latent variable models such as the Variational Auto-Encoder (VAE) have become
  a go-to tool for analyzing biological data, especially in the field of single-cell
  genomics. One remaining challenge is the interpretability of latent variables as
  biological processes that define a cellâ€™s identity. Outside of biological applications,
  this problem is commonly referred to as learning disentangled representations. Although
  several disentanglement-promoting variants of the VAE were introduced, and applied
  to single-cell genomics data, this task has been shown to be infeasible from independent
  and identically distributed measurements, without additional structure. Instead,
  recent methods propose to leverage non-stationary data, as well as the sparse mechanism
  shift assumption in order to learn disentangled representations with a causal semantic.
  Here, we extend the application of these methodological advances to the analysis
  of single-cell genomics data with genetic or chemical perturbations. More precisely,
  we propose a deep generative model of single-cell gene expression data for which
  each perturbation is treated as a stochastic intervention targeting an unknown,
  but sparse, subset of latent variables. We benchmark these methods on simulated
  single-cell data to evaluate their performance at latent units recovery, causal
  target identification and out-of-domain generalization. Finally, we apply those
  approaches to two real-world large-scale gene perturbation data sets and find that
  models that exploit the sparse mechanism shift hypothesis surpass contemporary methods
  on a transfer learning task. We implement our new model and benchmarks using the
  scvi-tools library, and release it as open-source software at \url{https://github.com/Genentech/sVAE}.
layout: inproceedings
series: Proceedings of Machine Learning Research
publisher: PMLR
issn: 2640-3498
id: lopez23a
month: 0
tex_title: Learning Causal Representations of Single Cells via Sparse Mechanism Shift
  Modeling
firstpage: 662
lastpage: 691
page: 662-691
order: 662
cycles: false
bibtex_author: Lopez, Romain and Tagasovska, Natasa and Ra, Stephen and Cho, Kyunghyun
  and Pritchard, Jonathan and Regev, Aviv
author:
- given: Romain
  family: Lopez
- given: Natasa
  family: Tagasovska
- given: Stephen
  family: Ra
- given: Kyunghyun
  family: Cho
- given: Jonathan
  family: Pritchard
- given: Aviv
  family: Regev
date: 2023-08-10
address:
container-title: Proceedings of the Second Conference on Causal Learning and Reasoning
volume: '213'
genre: inproceedings
issued:
  date-parts:
  - 2023
  - 8
  - 10
pdf: https://proceedings.mlr.press/v213/lopez23a/lopez23a.pdf
extras: []
# Format based on Martin Fenner's citeproc: https://blog.front-matter.io/posts/citeproc-yaml-for-bibliographies/
---
